modeling/inference_models/generic_hf_torch.py

```diff
@@ -3,6 +3,10 @@
 import os
 import json
 import torch
+try:
+    import intel_extension_for_pytorch as ipex
```

Comment:
`ipex` doesnt seem to be used directly in this file

Reply:
XPU support and torch.xpu is added by intel_extension_for_pytorch.